src/app/public/modules/split-view/split-view-workspace.component.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  Input
+} from '@angular/core';
+
+import {
+  SkyCoreAdapterService,
+  SkyMediaQueryService
+} from '@skyux/core';
+
+import {
+  SkySplitViewMediaQueryService
+} from './split-view-media-query.service';
+
+@Component({
+  selector: 'sky-split-view-workspace',
+  template: `<ng-content></ng-content>`,
```

Comment:
What's the benefit of putting the workspace and list logic in the split-view component, versus in their respective component classes? Seems like the list and workspace components should handle their own logic/template, if possible?

Reply:
Two major reasons:
1. These components are just shells for their respective `ng-content`.
2. The only logic performed on these components is updating their breakpoints. While that logic lives in each individual component (see `updateBreakpoint()), the parent split view component has to watch for changes to window resizing and consumers updating `listWidth`, and then call `updateBreakpoint()` on each child component to coordinate appropriate breakpoint changes to the DOM. At the time, a central authority on breakpoint updates seemed more clear than having that logic broken out an replicated across the two children components.

What do you think?

Reply:
That makes sense, thanks. At any rate, we can always change the behavior in the future, if needed.